include/pybind11/numpy.h

```diff
@@ -790,6 +795,15 @@ class array : public buffer {
         if (isinstance<array>(new_array)) { *this = std::move(new_array); }
     }
 
+    array reshape(ShapeContainer new_shape) {
```

Comment:
while we're at it, I'd also plumb through the `order`.
even if it's not actually used now (idk), it may be in future numpy versions.
easier to just do it than to explain why not.

Reply:
I think the opposite and it may have been used in older versions and its kept on for backwards compat.

Reply:
@rwgk We would need to add support for it as a function arg, which we currently don't. That can be added in a followup PR.

Reply:
> That can be added in a followup PR.
OK

Reply:
There is an NORDER Enum that it takes, but I am not sure how to expose that properly.

Reply:
sounds good, thanks for looking into it!